### ISSUE TITLE
dynamic module does not define module export function (PyInit_MalmoPython)

### ISSUE THREAD

**Issue by douglasrizzo** [Microsoft/malmo#738]
Hi everyone. I have installed the `malmo` Python package that comes with the repository (inside `scripts/python-wheel`). However, instead of installing it using [`linux_macos_wheel.sh`](https://github.com/Microsoft/malmo/blob/master/scripts/python-wheel/linux_macos_wheel.sh), which was not working, I had to create a small variant like so:

```
cp ../../build/install/Python_Examples/MalmoPython.so package/malmo
cp ../../Malmo/samples/Python_examples/malmoutils.py package/malmo
cp ../../build/install/Python_Examples/run_mission.py package/malmo
cp ../../Minecraft/launch_minecraft_in_background.py package/malmo
cd package
sudo -H python3 setup.py install
```

At this point, when I start `python3` and try to `import malmo.MalmoPython` I get the following error:

> dynamic module does not define module export function (PyInit_MalmoPython)

Other modules that try to `import MalmoPython` from inside `malmo` throw the following error: 

> ModuleNotFoundError: No module named 'MalmoPython'

Can anyone help me? Maybe I did something different when installing my package.

**Comment by AndKram** [comment]
The scripts/python-wheel directory is for maintainers creating the downloadable packages that can be installed from pypi.org by users doing a: "pip3 install malmo" (for Windows, Linux and MacOS with the latest version of Python 3 installed).  You should not need to build the wheel yourself. But you can of course do so and then install it locally as you say (or do a 'pip3 install -f "dist/*"' in the package dir).

Full instructions on the pip package are in: https://github.com/Microsoft/malmo/blob/master/scripts/python-wheel/README.md

**Comment by AndKram** [comment]
There is actually someone looking at building more wheels so you might be in luck soon.

**Comment by douglasrizzo** [comment]
Ubuntu 18.04 also comes with Python 3.6 as the default Python interpreter, so that will be necessary in the future. Is there no way I can build my own wheel? I imagine the problem is that MalmoPython.so comes pre-compiled, maybe I could compile it myself.

**Comment by douglasrizzo** [comment]
I think I can say with a fair degree of certainty that the problem was indeed my version of Python. I just installed the [`python35`](https://aur.archlinux.org/packages/python35/) package from AUR and I was able to run Malmo.

For other people who may want to run the software in Arch-based systems, you'll also need to downgrade the [boost](https://archive.archlinux.org/packages/b/boost/) and [boost-libs](https://archive.archlinux.org/packages/b/boost-libs/) packages to version 1.66, which are available in the Arch Linux archives. Version 1.66-0-3 of both packages worked for me.
